modules/era/sql/era_droplist.sql

```diff
@@ -6924,9 +6923,9 @@ INSERT INTO `mob_droplist` VALUES (825,0,0,1000,18597,@UNCOMMON); -- Catalyst (U
 
 -- ZoneID:  92 - Ferroalloy Quadav
 -- ZoneID: 200 - Fetid Flesh
-INSERT INTO `mob_droplist` VALUES (826,0,0,1000,849,@COMMON); -- Undead Skin (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (826,0,0,1000,940,@COMMON); -- Revival Tree Root (Common, 15%)
-INSERT INTO `mob_droplist` VALUES (826,0,0,1000,1041,@RARE);  -- Garlaige Chest Key (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (826,0,0,1000,940,@VCOMMON); -- Revival Tree Root (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (826,0,0,1000,849,@COMMON);  -- Undead Skin (Common, 15%)
+INSERT INTO `mob_droplist` VALUES (826,0,0,1000,1041,@VRARE);  -- Garlaige Chest Key (Very Rare, 1%)
```

Comment:
http://ffxidb.com/zones/200/fetid-flesh

Maybe keep this at 5%? There are only a  handful in the zone. Data points are pretty fucke because low data. 5% seems to be more in line with keys. Up to you